app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+$(document).ready(function() {
+    // Comments sending
+    $(".new_comment").on("ajax:success", function(e, data, status, xhr) {
+        flash_comment_result ('Коментар надіслано', 'succes');
+        this.reset();
+        remove_error_status(this);
+        update_comment_list(xhr.responseText);
+    }).on("ajax:error", function(e, xhr, status, error) {
+        flash_comment_result('Коментар не надіслано', "error");
+        remove_error_status(this);
+        render_errors(xhr.responseText, this);
+    });
+    // end of comments sending
+
+    // removing comments
+    $(".comments_list_wrapper").on("click", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+        if(!e.target.classList.contains('remove_comment')) return;
+        let request = remove_comment_request(e);
+        request.done(add_removed_status(e.target))
+    });
+    // end of removing comments
+});
+
+// Helpers
+function render_errors(json, form) {
+    let errors = JSON.parse(json);
```

Comment:
let? I'm afraid it might not work in some browsers. 
REMOVE all ecmascript 6 code from here, because lot's of code is not supported (let, for) https://kangax.github.io/compat-table/es6/
use normal JS